src/database/migrations/20241127141243-create-org-configs-table.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('organization_configs', {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER,
+			},
+			organization_id: {
+				allowNull: false,
+				primaryKey: true,
+				type: Sequelize.STRING,
+			},
+			data_manager_roles: {
```

Comment:
@adithyadinesh0412 let's call this column meta and its of type JSON, one of the key is data_manager_roles